packages/r/rax.json

```
@@ -0,0 +1,32 @@
+{
+  "name": "rax",
+  "filename": "rax.min.js",
+  "description": "A universal React-compatible render engine.",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/alibaba/rax"
+  },
+  "keywords": [
+    "rax",
+    "react",
+    "jsx"
+  ],
+  "homepage": "https://rax.js.org/",
+  "autoupdate": {
+    "source": "npm",
+    "target": "rax",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

```suggestion
          "*.js?(.map)"
```
cdnjs doesn't support hosting `.mjs` files, so we should only include the cjs & umd bundled files (and their maps).